v0/pipeline/steps/cd/merge-pr-step-info.yaml

```diff
@@ -17,6 +17,10 @@ allOf:
       oneOf:
       - type: boolean
       - type: string
+    disableGitRestraint:
+      oneOf:
+        - type: boolean
+        - type: string
```

Comment:
Ideally this shouldn't be just string but
```
- type: string
   pattern: (<\+.+>.*)
   minLength: 1
```

Reply:
updated